src/app/components/webauthn/login.tsx

```diff
@@ -52,6 +52,10 @@ function convertASN1toRaw(signatureBuffer: ArrayBuffer) {
     const sStart = usignature[rEnd + 2] === 0 ? rEnd + 3 : rEnd + 2;
     const r = usignature.slice(rStart, rEnd);
     const s = usignature.slice(sStart);
+    if (s[0] > 0x80) {
+        // TODO if is greater than secp256r1n/2 we need to flip s := secp256r1n - s
+        throw new Error('BAD SIGNATURE');
+    }
```

Comment:
Wow, what a coincidence, we just investigated why we have to flip , just few weeks ago.

Reply:
Exactly.